packages/block-library/src/embed/edit.js

```diff
@@ -179,9 +180,9 @@ const EmbedEdit = ( props ) => {
 
 	if ( fetching ) {
 		return (
-			<div { ...blockProps }>
+			<EmbedWrapper { ...blockProps }>
```

Comment:
Instead of introducing the `EmbedWrapper` here what do you think about utilizing the `View` [primitive](https://github.com/WordPress/gutenberg/tree/trunk/packages/primitives/src/view) that accomplishes this cross-platform behavior as well?
Let me know if this option is compatible here. Thanks!

Reply:
Didn't know about that one 😮 Seems perfect for this case!